elyra/pipeline/kfp/component_parser_kfp.py

```diff
@@ -87,9 +87,10 @@ def _parse_properties(self, component_yaml: Dict[str, Any]) -> List[ComponentPar
                 # Change type to reflect the parameter type (inputValue vs inputPath vs outputPath)
                 data_type_adjusted = data_type_parsed
                 if self._is_path_based_parameter(param.get('name'), component_yaml):
-                    data_type_adjusted = f"{param_type[:-1]}Path"
+                    data_type_adjusted = f"{param_type[:-1]}Path"  # i.e. inputpath to inputPath
```

Comment:
I don't understand how the comment relates to the statement.  I assume, from the comment that the value of `{param_type[:-1]}` must be the string `input`, but I don't understand the `inputPath to inputPath` - which implies some kind of transfer or conversion.

Reply:
After re-familiarizing myself with what's happening here, maybe the following comment makes more sense:

`# define adjusted type as either inputPath or outputPath`

Reply:
updated